lattice/cpp/header_entries.py

```diff
@@ -139,7 +141,9 @@ def __str__(self):
         entry = f"{self._indent}{self.type} {self.name} {self._opener}\n"
         for e in self._enumerants:
             entry += f"{self._indent}\t{e},\n"
-        entry += f"{self._indent}\tUNKNOWN\n{self._indent}{self._closure}"
+        if "UNKNOWN" not in self._enumerants:
+            entry += f"{self._indent}\tUNKNOWN\n"
```

Comment:
ClimateInformation::DataSourceType already has an UNKNOWN enumerant in the schema. We've been adding one in all the other enums just for the cpp code. So, now we check first.

Reply:
hmmm...do we remember why? This might be worth looking into more later. We can make an issue for now.

Reply:
My answer is usually, "Chip".